### PR TITLE
fix(i18n): update Chinese translation of basic html and html5

### PR DESCRIPTION
**Pull request by ZhichengChen**
- [X] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] None of my changes are plagiarized from another source without proper attribution.
- [X] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [X] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md [hunk 1]
```diff
@@ -2,29 +2,36 @@
...
+forumTopicId: 16627
+localeTitle: 给表单添加提交按钮
 ---
 
 ## Description
-<section id="description">我们在表单中添加一个<code>submit</code>按钮。单击此按钮会将表单中的数据发送到您使用表单的<code>action</code>属性指定的URL。这是一个示例提交按钮： <code>&lt;button type=&quot;submit&quot;&gt;this button submits the form&lt;/button&gt;</code> </section>
+<section id='description'>
+让我们来给表单添加一个<code>submit</code>提交按钮，当点击提交按钮时，表单中的数据将会被发送到<code>action</code>属性指定的地址上。
+例如：
+<code>&#60;button type="submit"&#62;this button submits the form&#60;/button&#62;</code>
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md]: @raisedadead Should we translate the example snippet into Chinese as well (More specifically, "this button submits the form" here)? Personally, I'd consider that makes more sense by having it translated.

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md [hunk 2]
```diff
@@ -38,61 +45,32 @@ tests:
 ```html
 <h2>CatPhotoApp</h2>
 <main>
-  <p>Click here to view more <a href="#">cat photos</a>.</p>
-
-  <a href="#"><img src="https://bit.ly/fcc-relaxing-cat" alt="A cute orange cat lying on its back."></a>
-
-  <p>Things cats love:</p>
+<p>点击查看更多<a href="#">猫咪图片</a>。</p>
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md]: nit: incorrect indent

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md [hunk 3]
```diff
@@ -38,61 +45,32 @@ tests:
...
+  
+  <a href="#"><img src="https://bit.ly/fcc-relaxing-cat" alt="一只仰卧着的萌猫"></a>
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md]: orange cat: 橘猫

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md [hunk 4]
```diff
@@ -38,61 +45,32 @@ tests:
...
+  
+  <p>猫咪最喜欢的三件东西：</p>
   <ul>
-    <li>cat nip</li>
-    <li>laser pointers</li>
-    <li>lasagna</li>
+    <li>猫薄荷</li>
+    <li>激光笔</li>
+    <li>千层饼</li>
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md]: Lasagna: [千层面](https://zh.wikipedia.org/wiki/%E5%8D%83%E5%B1%A4%E9%BA%B5)。这个梗的来源是[加菲猫](https://zh.wikipedia.org/wiki/%E5%8A%A0%E8%8F%B2%E7%8C%AB)

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md [hunk 5]
```diff
@@ -38,61 +45,32 @@ tests:
...
   </ul>
-  <p>Top 3 things cats hate:</p>
+  <p>猫咪最讨厌的三件东西：</p>
   <ol>
-    <li>flea treatment</li>
-    <li>thunder</li>
-    <li>other cats</li>
+    <li>跳蚤</li>
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md]: flea 是“跳蚤”，但 flea treatment 是“除跳蚤”或者“祛跳蚤”。你选一个吧😂

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md [hunk 6]
```diff
@@ -2,29 +2,36 @@
...
-localeTitle: 向表单添加提交按钮
+videoUrl: 'https://scrimba.com/p/pVMPUv/cp2Nkhz'
+forumTopicId: 16627
+localeTitle: 给表单添加提交按钮
 ---
 
 ## Description
-<section id="description">我们在表单中添加一个<code>submit</code>按钮。单击此按钮会将表单中的数据发送到您使用表单的<code>action</code>属性指定的URL。这是一个示例提交按钮： <code>&lt;button type=&quot;submit&quot;&gt;this button submits the form&lt;/button&gt;</code> </section>
+<section id='description'>
+让我们来给表单添加一个<code>submit</code>提交按钮，当点击提交按钮时，表单中的数据将会被发送到<code>action</code>属性指定的地址上。
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-a-submit-button-to-a-form.chinese.md]: 我觉得这里用 `URL` 会比 "地址" 更好。你觉得呢？

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.chinese.md [hunk 7]
```diff
@@ -2,27 +2,44 @@
...
 ## Description
-<section id="description">您可以使用<code>img</code>元素将图像添加到网站，并使用<code>src</code>属性指向特定图像的URL。这方面的一个例子是： <code>&lt;img src=&quot;https://www.your-image-source.com/your-image.jpg&quot;&gt;</code>请注意， <code>img</code>元素是自动关闭的。所有<code>img</code>元素都<strong>必须</strong>具有<code>alt</code>属性。 <code>alt</code>属性中的文本用于屏幕阅读器以提高可访问性，并在图像无法加载时显示。注意：如果图像纯粹是装饰性的，则使用空的<code>alt</code>属性是最佳做法。理想情况下，除非需要，否则<code>alt</code>属性不应包含特殊字符。让我们在上面的<code>img</code>示例中添加一个<code>alt</code>属性： <code>&lt;img src=&quot;https://www.your-image-source.com/your-image.jpg&quot; alt=&quot;Author standing on a beach with two thumbs up.&quot;&gt;</code> </section>
+<section id='description'>
+用<code>img</code>元素来为你的网站添加图片，其中<code>src</code>属性指向一个图片的地址。
+例如：
+<code>&#60img src="https://www.your-image-source.com/your-image.jpg"&#62</code>
+注意：<code>img</code>元素是没有结束标记的。
+所有的<code>img</code>元素必须有<code>alt</code>属性，<code>alt</code>属性的文本是当图片无法加载时显示的替代文本，这对于通过屏幕阅读器来浏览网页的用户非常重要。
+注意：如果图片是纯装饰性的，用一个空的<code>alt</code>是最佳实践。
+理想情况下，<code>alt</code>属性不应该包含特殊字符，除非需要。
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.chinese.md]: 除非需要 => 除非必要

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.chinese.md [hunk 8]
```diff
@@ -2,27 +2,44 @@
...
 ```yml
 tests:
-  - text: 您的页面应该有一个图像元素。
-    testString: 'assert($("img").length > 0, "Your page should have an image element.");'
-  - text: 您的图像应具有指向小猫图像的<code>src</code>属性。
-    testString: 'assert(new RegExp("\/\/bit.ly\/fcc-relaxing-cat|\/\/s3.amazonaws.com\/freecodecamp\/relaxing-cat.jpg", "gi").test($("img").attr("src")), "Your image should have a <code>src</code> attribute that points to the kitten image.");'
-  - text: 您的图片元素<strong>必须</strong>具有<code>alt</code>属性。
-    testString: 'assert(code.match(/alt\s*?=\s*?(\"|\").*(\"|\")/), "Your image element <strong>must</strong> have an <code>alt</code> attribute.");'
+  - text: '网页应该有一张图片。'
+    testString: assert($("img").length > 0);
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.chinese.md]: `assert` 的第二个参数，印象中是 failure 的时候显示的内容。应该也需要翻译吧？和上面一样就好

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.chinese.md [hunk 9]
```diff
@@ -2,27 +2,44 @@
...
+  - text: '这张图片应该是一只小猫。'
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.chinese.md]: 这个描述我觉得可能不够具体，因为 test 内容是 `img` 的 `src`。你觉得有必要描述得详细一些么？

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.chinese.md [hunk 10]
```diff
@@ -36,12 +53,11 @@ tests:
...
 <h2>CatPhotoApp</h2>
 <main>
-
-
-  <p>Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>
-  <p>Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>
+  
+  
+  <p>在大家心目中，猫是慵懒和可爱的化身，它可以睡饱了再起来吃饭，可以逗趣小耗子，可以卖得了萌，使得了坏，这样百变的小怪兽就集结在一只宠物上，怎能不惹人怜爱。</p>
+  <p>养猫有的时候，就是介于爱与恨之间，当你钦羡别人萌宠这么可爱的时候，你一定没有想过，猫咪会到处掉毛，甚至会屯老鼠，啃鞋子，用爪子爬门，你不理它，它就挠你，你要对它发脾气，它会比你更来劲。所以，猫咪慎入，没有一定的准备，切勿随便去侍养动物。它们一旦认定你了，你就是它们的主人，如果你抛弃它们，它们必定心中重创。</p>
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.chinese.md]: 应该是“囤老鼠”吧？
另外，"猫咪慎入，"感觉可以去掉

**Review by S1ngS1ng — CHANGES_REQUESTED** [review]
@raisedadead Here are some questions to you:
1. Should we translate the text in the code sample as well? (IMO, it is better to have them translated).
2. I noticed that all the contents of `Solution` have been removed. Is that as planned?
3. Looks like the single quotes of `Tests/tests/textString` have been removed, while a pair of single quotes have been added to `Tests/tests/text`. Is it fine?

@ZhichengChen Just reviewed the first file. Will review the rest tomorrow.

**Comment by raisedadead** [comment]
> @raisedadead Here are some questions to you:
> 
> 1. Should we translate the text in the code sample as well? (IMO, it is better to have them translated).
> 2. I noticed that all the contents of `Solution` have been removed. Is that as planned?
> 3. Looks like the single quotes of `Tests/tests/textString` have been removed, while a pair of single quotes have been added to `Tests/tests/text`. Is it fine?
> 

Hi @RandellDawson can you guide us here when you get a minute.

**Comment by RandellDawson** [comment]
> Should we translate the text in the code sample as well? (IMO, it is better to have them translated).

If you do it there, will you be translating text in the actually challenge seed code?  If so, you may also have to update the tests that may be testing for a specific string.  @raisedadead I personally think the code text should remain in English and then we figure out a way to leverage the solution contained in the English version to use as the solution for each language.  This way, the only thing the `textString` would not have to change at all and only the `text` would be translated where applicable.

> I noticed that all the contents of Solution have been removed. Is that as planned?

If there are solutions missing, I am not sure that was intentional.  However, my response above is still my recommendation for the solutions across all languages.

> Looks like the single quotes of Tests/tests/textString have been removed, while a pair of single quotes have been added to Tests/tests/text. Is it fine?

If you take a look at the English versions of the same challenge (which you should do anything to validate the translation is correct and there are not missing or changed `text` or `testString`), you will see how to properly format each.  

You will also note that the English version for all but the back-end challenges and React challenges does not include text for an assert message.  That is because the `text` is used for the assert message.  As long as you are using the English version as a guide for each challenge, you can see when to include them and when to not include them.

@ojeytonwilliams Any thoughts on how we can parse the English solution(s) out to use for each language?  This would simplify the non-English versions and keep them on track with the English version of the challenge.z

@raisedadead If we decide to let each language stand alone (which has also been discussed in the past), then the solutions would be unique to each language and then it would make sense to allow the translation of the text in code and the challenge seed and solutions.  It is really your call.  I have already stated my preference.

**Comment by raisedadead** [comment]
I think the seed code should be in english. We should be able to use the script that you have to populate the Chinese curriculum after these PRs have landed.

**Comment by ojeytonwilliams** [comment]
> Any thoughts on how we can parse the English solution(s) out to use for each language? This would simplify the non-English versions and keep them on track with the English version of the challenge.

What's the objective?  Extract the English solutions and overwrite the solutions for the other languages?  It sounds like you already have a script for that, so I assume I'm missing something.

**Comment by RandellDawson** [comment]
@raisedadead If the seed code is in English, then if we parse out the solution(s) from the English version of the same challenge, then the testStrings would not need translation nor would we need to copy over the solutions.

**Comment by RandellDawson** [comment]
> What's the objective? Extract the English solutions and overwrite the solutions for the other languages? It sounds like you already have a script for that, so I assume I'm missing something.

No, I mean the non-English files would not have a solutions section or it could be empty because the challenge parser would get them from the English version.

**Comment by raisedadead** [comment]
You are correct, the parser logic would need some work.

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.chinese.md (hunk 8 above)
Comment (ZhichengChen) [reply curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-images-to-your-website.chinese.md]: 在英文的最新的版本里去掉了第二个参数，这里是个冗余参数，我在本地测试过，没有第二个参数也是 OK 的

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/uncomment-html.chinese.md [hunk 11]
```diff
@@ -41,9 +48,8 @@ tests:
 
 <h2>CatPhotoApp</h2>
 
-<p>Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>
+<p>Monkey code 猴哥猴哥，你真了不得，金箍棒在手，问世间谁是英雄。</p>
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/uncomment-html.chinese.md]: 上午讨论的，如果之前那道题目要改，这里也得同步

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-html5-to-require-a-field.chinese.md [hunk 12]
```diff
@@ -2,23 +2,30 @@
...
+videoUrl: 'https://scrimba.com/p/pVMPUv/cMd4EcQ'
+forumTopicId: 18360
+localeTitle: 给表单添加一个必填字段
 ---
 
 ## Description
-<section id="description">您可以订定必填项，限制您的用户在完成必填项之前无法提交表单。如果您要订定必填项段，只需在<code>input</code>元素中添加<code>required</code>的属性，如下所示： <code>&lt;input type=&quot;text&quot; required&gt;</code> </section>
+<section id='description'>
+当你设计表单时，你可以指定某些字段为必填项(required)，只有当用户填写了该字段后，才可以提交表单。
+如果你想把文本输入框设置为必填项，在<code>input</code>元素中加上required属性就可以了，例如：<code>&#60;input type="text" required&#62;</code>
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-html5-to-require-a-field.chinese.md]: “加上required属性”
required 两边应该有空格

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-html5-to-require-a-field.chinese.md [hunk 13]
```diff
@@ -2,23 +2,30 @@
...
+</section>
 
 ## Instructions
-<section id="instructions">利用<code>required</code> 使您的<code>input</code>成为必填项 ，限制您的用户在沒填写必填项的情况下无法提交表单。请尝试在沒有在完成必填项前提交表单，看看HTML5如何通知您少填了必填项。 </section>
+<section id='instructions'>
+确保<code>input</code>元素中有<code>required</code>属性，否则无法提交表单。
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-html5-to-require-a-field.chinese.md]: 给\<code>input\</code>元素加上\<code>required\</code>属性，这样用户就必须先在输入框里填入内容，然后才可以提交表单。

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.chinese.md [hunk 14]
```diff
@@ -0,0 +1,130 @@
...
+id: 5c6c06847491271903d37cfd
+title: Use the value attribute with Radio Buttons and Checkboxes
+challengeType: 0
+forumTopicId: 301099
+localeTitle: 使用单选框和复选框的 value 属性
+---
+
+## Description
+<section id='description'>
+当表单提交时，所有的数据会发送到服务端。包括 options 的已选值。<code>radio</code> 和 <code>checkbox</code> 取其 <code>value</code> 属性的值。
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.chinese.md]: 当表单提交时，包括 options 已选值在内的数据会发送给服务端。<code>radio</code>和<code>checkbox</code>的<code>value</code>值决定了发送到服务端的实际内容。

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.chinese.md [hunk 15]
```diff
@@ -0,0 +1,130 @@
...
+  <input id="indoor" value="indoor" type="radio" name="indoor-outdoor">Indoor 
+</label>
+<label for="outdoor"> 
+  <input id="outdoor" value="outdoor" type="radio" name="indoor-outdoor">Outdoor 
+</label>
+```
+
+在这里，有两个 <code>radio</code> 单选框。如果当用户提交表单时 <code>indoor</code>  选项被选中，表单数据会包含：<code>indoor-outdoor=indoor</code>。也就是 "indoor" 单选框的 <code>name</code> 和 <code>value</code> 属性。
+
+如果没写 <code>value</code> 属性，会使用默认值做为表单数据提交，也就是 <code>on</code>。在这种情况下，如果用户点击 "indoor" 选项然后提交表单，表单数据的值为 <code>indoor-outdoor=on</code>，这可能并没有什么意义。因此最好将 <code>value</code> 属性设置一些有意义的内容。
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.chinese.md]: \<code> 标签周围应该没有空格吧？下同

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.chinese.md [hunk 16]
```diff
@@ -0,0 +1,130 @@
...
+</section>
+
+## Instructions
+<section id='instructions'>
+给每一个 <code>radio</code> 和 <code>checkbox</code> 输入框指定 <code>value</code> 属性。使用 input 标签文字，转换为小写，作为 value 的值。
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.chinese.md]: 给每一个\<code>radio\</code>和\<code>checkbox\</code>输入框添加\<code>value\</code>属性。请把每个\<code>input\</code>对应的\<code>label\</code>文本转换为小写（如 Outdoor 应转换为 outdoor），设置其为 value 的值（即 \<code>value="outdoor"\</code>）。

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.chinese.md [hunk 17]
```diff
@@ -0,0 +1,130 @@
...
+    <label><input type="checkbox" name="personality" value="loving"> Loving</label>
+    <label><input type="checkbox" name="personality" value="lazy"> Lazy</label>
+    <label><input type="checkbox" name="personality" value="energetic"> Energetic</label><br>
+    <input type="text" placeholder="cat photo URL" required>
+    <button type="submit">Submit</button>
+  </form>
+</main>
+```
+
+</section>
```
Comment (S1ngS1ng) [inline curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.chinese.md]: 这部分是不是应该去掉？

**Review by S1ngS1ng — CHANGES_REQUESTED** [review]
@ZhichengChen 
1. 建议保留 \<code>a\</code>，而不翻译为“锚点”。see: https://developer.mozilla.org/zh-CN/docs/Web/HTML/Element/a 。如果要改，记得全局查找替换下，测试文本里也有
2. 有几个地方的空格需要调整下

其他没啥问题

@raisedadead 90% of **THIS** PR is good to go. Still, there are some nitpicks to be fixed. Should be ready in a day or two.
I synced with @ZhichengChen this morning. We've decided to collaborate on the translation both "online" and "offline". 
The "online" refers to the ~~conventional~~ or ~~canonical~~ way that we bot post our review comments here at GitHub. Hopefully, you guys are fine when we post the review comments in Chinese, since this makes our review process way easier. We'll make sure to post the comments in English when we consider it related to other members/contributors.
The "offline" works in a way that we both hop over on a session (e.g. Hangout, Zoom.us) to review the PR, which should be ready to ship right after. Due to the timezone, this may happen once/twice a week. 

Since the review process could take a bit while, hopefully, it won't block the release of the next version.

In case of any questions/comments/concerns, please let us know. Thanks!

File: curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.chinese.md (hunk 15 above)
Comment (ZhichengChen) [reply curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.chinese.md]: 这个地方我昨天忘记和你说了，新版 css 做了调整，<code> 标签周围是有空格的，
https://raw.githubusercontent.com/freeCodeCamp/freeCodeCamp/master/curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.english.md
个人觉得可以添加空格，这样不管样式如何改变，显示效果不会有太大的影响。
另外好多 testString 上面也有 <code> 标签，如果不加空格的话显示会有点问题。
另外 <code> 标签页大部分没有添加空格，数目比较大，我觉得可以优先级调低。后期在慢慢统一。

Comment (S1ngS1ng) [reply curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes.chinese.md]: @ZhichengChen 好的，没问题

**Review by S1ngS1ng — APPROVED** [review]
LGTM 👍

**Review by S1ngS1ng — APPROVED** [review]
LGTM 👍

**Comment by ZhichengChen** [comment]
@S1ngS1ng need approve to merge

**Review by raisedadead — APPROVED** [review]

**Pull request merged**